hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -229,15 +229,19 @@ def on_print_call_stack(sim: OTBNSim, args: List[str]) -> None:
     for value in sim.state.peek_call_stack():
         print('0x{:08x}'.format(value))
 
-
-def on_edn_rnd_data(sim: OTBNSim, args: List[str]) -> None:
-    if len(args) != 1:
-        raise ValueError('edn_rnd_data expects exactly 1 argument. Got {}.'
+def on_edn_step(sim: OTBNSim, args: List[str]) -> None:
+    if len(args) != 2:
+        raise ValueError('edn_step expects exactly 2 argument. Got {}.'
                          .format(args))
 
-    edn_rnd_data = read_word('edn_rnd_data', args[0], 256)
-    sim.state.set_rnd_data(edn_rnd_data)
+    edn_rnd_data = read_word('edn_step', args[0], 32)
+    edn_rnd_data_valid = bool(args[1])
+    edn_rnd_data_valid = int(args[1])
 
+    if edn_rnd_data_valid:
+        sim.state.rnd_completed()
+    else:
+        sim.state.step_edn(edn_rnd_data)
```

Comment:
I'm not sure this correctly deals with a cycle where `edn_rnd_data_ack && edn_rnd_data_valid`. I don't *think* that can ever happen in the design, but could we put a check somewhere to make sure? (Possibly it belongs on the C++ side, or maybe we should pass the ack flag through to the Python?)